docs/our_work/ds218_rap_community_of_practice.md

```diff
@@ -1,29 +1,42 @@
 ---
 layout: base
 title: NHS Reproducible Analytical Pipelines (RAP) Champion Squad
-permalink: ds218_rap_community_of_practice.html
+summary: Our roving squad of RAP champions have helped a number of teams not only transform their pipelines, but also taught them how to train others and produced guidance which is used by many other organisations.
+origin: 'NHS Digital'
+tags: ['RAP','Best Practice','Python', 'R']
 ---
 
-> | "Our roving squad of RAP champions have helped a number of teams not only transform their pipelines, but also tought them how to train others and produced guidance which is used by many other organisations."   
-
-<a href = "https://nhsdigital.github.io/rap-community-of-practice/">
-<figure markdown >
-![An image displaying the front page of the NHS RAP Community of Practice website.](../images/ds218_rap_community_of_practice.png) </a>
-<figcaption>Figure 1: The front page of the RAP Community of Practice website.</figcaption>
+<a href="https://www.linkedin.com/feed/hashtag/reproducibleanalyticalpipelines/">
```

Comment:
how come you want to link to linked in instead of the RAP website? or should we link to both?

Reply:
ah doesn't the page a link to the website? I'll have a look and perhaps reconsider!

Reply:
changed!